src/login/AzureLoginHelper.ts

```diff
@@ -411,11 +411,15 @@ export class AzureLoginHelper {
 		});
 	}
 
-	private async noSubscriptionsFound(): Promise<void> {
-		const open: MessageItem = { title: localize('azure-account.open', "Open") };
-		const response: MessageItem | undefined = await window.showInformationMessage(localize('azure-account.noSubscriptionsFound', "No subscriptions were found. Set up your account at https://azure.microsoft.com/en-us/free/."), open);
-		if (response === open) {
-			void openUri('https://azure.microsoft.com/en-us/free/?utm_source=campaign&utm_campaign=vscode-azure-account&mktingSource=vscode-azure-account');
+	private async showNoSubscriptionsFoundNotification(): Promise<void> {
+		const noSubscriptionsFound = localize('azure-account.noSubscriptionsFound', 'No subscriptions were found. Check out our troubleshooting page for common solutions to this problem or setup your account.');
```

Comment:
Should we present setup as the first option or troubleshooting? I can see valid arguments both ways. @fiveisprime what do you think?

Either way, we should make the button order match the prose (as you have here).

Reply:
> Should we present setup as the first option or troubleshooting? I can see valid arguments both ways.

I agree - I could see it going either way. I'm cool with deferring to @fiveisprime on this

Reply:
@jinglouMSFT, @alexweininger, and I put on our PM hats and decided that it's better if we swap the message's order. Also, both links now take you to different headers in the [troubleshooting page](https://github.com/microsoft/vscode-azure-account/wiki/Troubleshooting) where I added some more details about setting up your account.